### ISSUE TITLE
Selection-based custom command doesn't realise a selection has been made

### ISSUE THREAD

**Issue by vadi2** [sourcegraph/cody#1539]
I defined the following custom command:

```json
{
  "commands": {
    "/rewrite-in-lua": {
      "prompt": "Take the currently opened contents and rewrite in Lua please",
      "context": {
        "codebase": false,
        "selection": true
      },
      "description": "Rewrite the selected contents in Lua"
    }
  }
}
```

When I use the custom command on a partial selection, it works fine. It does not work however when I use on the entire file - it thinks nothing has been selected:

[Screencast from 28-10-23 09:24:43.webm](https://github.com/sourcegraph/cody/assets/110988/98c557a6-3bac-4afb-8263-63fbe4a64044)

**Comment by elvisisvan** [comment]
have you tried manually typing `/rewrite-in-lua` and `ctrl + enter`?

**Comment by vadi2** [comment]
No luck, same thing:

> /rewrite-in-lua requires highlighted code. Please select some code in your editor and try again.

**Comment by arafatkatze** [comment]
@vadi2 Apologies for the issue. 

It has already been fixed by @abeatrix  here -> https://github.com/sourcegraph/cody/pull/1508  The latest version with the fix hasn't been released. If you select everything but the first line then the selections should work, the latest release of Cody shall fix this.

**Comment by jdorfman** [comment]
This will be published tomorrow. 🎉

**Comment by abeatrix** [comment]
@vadi2 The fix has just been released in 0.14.3, please let me know if that works for you :) 
![image](https://github.com/sourcegraph/cody/assets/68532117/597d5795-a089-49f1-937e-954d3ce2992f)

**Comment by vadi2** [comment]
It does! Thank you.
